framework/platform/bpl/CMakeLists.txt

```diff
@@ -8,6 +8,23 @@ set(MODULE_PATH ${CMAKE_CURRENT_LIST_DIR})
 # Common Sources
 file(GLOB_RECURSE bpl_common_sources ${MODULE_PATH}/common/*.c*)
 
+# Common prplmesh_db values
+set(BEEROCKS_MANAGEMENT_MODE  "Multi-AP-Controller-and-Agent" CACHE STRING "Set management mode")
+set(BEEROCKS_OPERATING_MODE  "Gateway" CACHE STRING "Set operating mode")
+set(BEEROCKS_STOP_ON_FAILURE_ATTEMPTS  0 CACHE INT "In common config stop on failure disabled")
+set(BEEROCKS_CERTIFICATION_MODE  1 CACHE INT "In certification, agent must retry onboarding indefinitely")
```

Comment:
These ones aren't used anywhere (and I don't think they should be).

Reply:
It is used in prplmesh_platform_db.in file